src/Raven.Server/ServerWide/Commands/UpdatePullReplicationAsSinkCommand.cs

```diff
@@ -29,6 +29,19 @@ public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
             }
             else
             {
+                // if new definition doesn't have certificate but there is old one with cert 
+                // it means we want to use existing cert
+                if (PullReplicationAsSink.CertificateWithPrivateKey == null)
```

Comment:
We will not be able to remove the cert once it was set, only override it. Just want a confirmation that this is what we wanted.

Reply:
we have 2 cases:
http -> you can't provide certificate
https -> you have to provide certificate

so validation guard against this. 

I've added this condition (and I'm not proud of it) but it allows us using same cert when doing edit.